src/main/service/listParseHelperService.ts

```diff
@@ -250,26 +251,35 @@ export class ListParseHelperService {
       sitting['durationAsMinutes'] = durationAsMinutes;
       sitting['durationAsDays'] = durationAsDays;
 
-      const min = moment(sitting['sittingStart'], 'HH:mm').minutes();
-      if (min === 0) {
-        sitting['time'] = moment.utc(sitting['sittingStart']).tz(this.timeZone).format('ha');
+      if (sittingStart.minute === 0) {
+        this.formatCaseTime(sitting, 'ha');
       } else {
-        sitting['time'] = moment.utc(sitting['sittingStart']).tz(this.timeZone).format('h:mma');
+        this.formatCaseTime(sitting, 'h:mma');
       }
     }
   }
 
+  public formatCaseTime(sitting: object, format: string): void {
+    if (sitting['sittingStart'] !== '') {
+      let zonedDateTime = DateTime.fromISO(sitting['sittingStart'], {zone: this.timeZone});
+      //Unable to find a way to convert time to BST. Luxon always return offset (+01:00) with the time,
```

Comment:
I've tried out a few lists for the Content Date, and it seems to work OK (In summer, adds one hour) with the normal DateTime.fromISO.

Wondering if the same could be done in this method rather than having to manually apply the offset

Reply:
As discussed, I am checking the Z at the end of time, if it is there, don't add offset to the time.